### ISSUE TITLE
Investigate getdevshop

### ISSUE THREAD

**Issue by btopro** [elmsln/spawn#8]
May not be The solution to what we're describing but should be something we check out as a more advanced version of the prototype we're talking about building. Spawn might be great for making site factories for less experienced site managers / site builders where as DevShop might be a better option for developers or for managing multiple site factories (like as our way of letting people kick out their own site factories)

**Comment by bradallenfisher** [comment]
good point. there's no reason you shouldn't be able to use spawn to spit out more spawns

**Comment by baf139** [comment]
good point.

**Comment by baf139** [comment]
looking at devshop it seems pretty cool. It's built on Aegir which is sort of how I envision SPAWN minus all the complications for RHEL based OS.

**Comment by btopro** [comment]
:) yeess.....
